.github/workflows/release_linux_i686.yml

```diff
@@ -7,11 +7,21 @@ on:
   push:
     branches: [master, rel-*]
   pull_request:
-    branches: [rel-*]
+    branches: [master, rel-*]
```

Comment:
why add master here?

Reply:
Add PR for master here to let the following code (L24) be able to consider which labels that PR is using. If the PR for master has a label "run release CIs", it will run whole release CIs; If not, it won't.